### PR TITLE
Added matomo events for GraphQL errors and Restrictions

### PR DESCRIPTION
**Pull request by svenjens**
Description 
Added extra matomo events. 

Please make sure:
- [x] to add a label
- [x] to add one or more reviewers
- [x] you followed our checklist for [functional testing](https://github.com/Amsterdam/vergunningcheck/blob/master/TESTING.md)
- [ ] you added the necessary documentation (either in the markdown files or inline)
- [ ] you added the necessary automated tests

### CODE REVIEW

**Comment by afjlambert** [comment]
@svenjens is this ready for review or not?

**Comment by robinpiets** [comment]
> @svenjens is this ready for review or not?

Yes, I've updated the PR @afjlambert

File: apps/client/src/components/Location/LocationInput.tsx [hunk 1]
```diff
@@ -61,6 +71,14 @@ const LocationInput: React.FC<{
...
         name: address.postalCode.substring(0, 4),
       });
+      matomoTrackEvent({
+        action: actions.SUBMIT_LOCATION,
+        name: monument ? monument : eventNames.NO_MONUMENT,
+      });
+      matomoTrackEvent({
+        action: actions.SUBMIT_LOCATION,
+        name: cityScape ? cityScape : eventNames.NO_CITYSCAPE,
+      });
```
Comment (afjlambert) [inline apps/client/src/components/Location/LocationInput.tsx]: Do we really need to send 3 separate events?
Is there no way to send it as 1 event or at least 1 network request?

File: apps/client/src/components/Location/LocationInput.tsx [hunk 2]
```diff
@@ -61,6 +71,14 @@ const LocationInput: React.FC<{
         action: actions.SUBMIT_LOCATION,
         name: address.postalCode.substring(0, 4),
       });
+      matomoTrackEvent({
+        action: actions.SUBMIT_LOCATION,
+        name: monument ? monument : eventNames.NO_MONUMENT,
```
Comment (afjlambert) [inline apps/client/src/components/Location/LocationInput.tsx]: Can you use the `||` operator here (and below)?

**Review by afjlambert — CHANGES_REQUESTED** [review]

File: apps/client/src/components/Location/LocationInput.tsx (hunk 2 above)
Comment (robinpiets) [reply apps/client/src/components/Location/LocationInput.tsx]: Fixed

File: apps/client/src/components/Location/LocationInput.tsx (hunk 1 above)
Comment (robinpiets) [reply apps/client/src/components/Location/LocationInput.tsx]: There is no way unfortunately

**Review by afjlambert — APPROVED** [review]

**Pull request merged**